api/.docker/deps_ok_then

```diff
@@ -2,7 +2,7 @@
 
 if [ ! -x node_modules/.bin/deps-ok ]; then
   echo "Installing deps-ok"
-  npm install --quiet deps-ok
+  npm install --save-dev --quiet deps-ok
```

Comment:
Sorry, why are we doing `--save-dev` now?

Reply:
The `deps-ok` lib (which confirms top-level modules are installed) is only used during development. Previously, we were running `npm install` _without_ `--save`, which would add the lib to our environment but not affect our package/shrinkwrap files. The new version of npm automatically updates these files, however, so we need to place it differently.